oar/cli/cmd_image_consistency_check.py

```diff
@@ -50,15 +50,15 @@ def image_consistency_check(ctx, build_number, for_nightly):
                     if for_nightly:
                         if "x86_64" in cs.get_candidate_builds():
                             pull_spec = (
-                                "registry.ci.openshift.org/ocp/release:" + cs.get_candidate_builds().get("x86_64") 
+                                "registry.ci.openshift.org/ocp/release: " + cs.get_candidate_builds().get("x86_64") 
                             )
                         else: 
                             raise JenkinsHelperException(
                                 f"no candidate nightly build for x86_64 architecture"
                                 )
                     else:
                         pull_spec = (
-                            "quay.io/openshift-release-dev/ocp-release:" + cs.release + "-x86_64" 
+                            "quay.io/openshift-release-dev/ocp-release: " + cs.release + "-x86_64" 
```

Comment:
space should not be there for image tag.
e.g. registry.ci.openshift.org/ocp/release:4.16.0-0.nightly-2024-09-16-095421

Reply:
Thanks Rio, I did this automatically and did not realize this. All issues should be fixed now